rundeckapp/grails-app/assets/javascripts/resourceModelConfig.js

```diff
@@ -44,7 +44,7 @@ var ResourceModelConfigControl = Class.create({
     }else{
         content.addClassName('rsrcConfigContent');
     }
-    $(content).select('input').each(function(elem) {
+    $(content).on('select','input').each(function(elem) {
```

Comment:
I think the prototype code `$(content).select('input')` should just be converted to `jQuery('input')`